### PR TITLE
Adding a file for Dublin NH

### LINKED ISSUE #26: jgboyd things to do
- [ ] Create an issue
- [ ] Create a branch
- [ ] Create commits
- [ ] Create a PR
- [ ] Collaborate
- [ ] Merge

### PR DESCRIPTION
**Pull request by jgboyd**
Pull request for adding a file with places to eat and things to do in Dublin NH. This PR resolves #26.

@groberge what would you do in manchester?

### CODE REVIEW

File: jgboyd-hometown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Dublin, NH
+
+### Great Places to Eat
```
Comment (jgboyd) [inline jgboyd-hometown.md]: Jim recommends Del Rossis :Del Rossis:

Comment (neuwib) [inline jgboyd-hometown.md]: Where can I go camping?

Comment (jgboyd) [inline jgboyd-hometown.md]: Mount Monadnock State Park.  Best in the area...

Comment (neuwib) [inline jgboyd-hometown.md]: Any good places to go hiking?

Comment (jgboyd) [inline jgboyd-hometown.md]: Same.  Mt Monadnock is a great hike.  You can see five states on a clear day and even Boston from the summit

**Pull request merged**